docs/standard/schema.core.rst

```diff
@@ -113,11 +113,11 @@ Key ``releaseDate``
 ~~~~~~~~~~~~~~~~~~~
 
 -  Type: string (date)
--  Presence: mandatory
+-  Presence: optional
 -  Example: ``"2017-04-15"``
 
 This key contains the date at which the latest version was released.
-This date is mandatory if the software has been released at least once
+This date is only mandatory if the software has been released at least once
 and thus the version number is present.
```

Comment:
```suggestion
```

Reply:
WDYT @tomootes?

Reply:
I agree. In the case of `softwareVersion: dev` its not logical to have a `releaseDate`. Always optional is the way to go then!

Reply:
@bfabio sorry for my late answer btw. I'm not an open source contributer regularly, YET haha.

Reply:
@tomootes don't worry, this is just your first contribution of many to come :wink: